src/scripts/chart/column.js

```diff
@@ -22,13 +24,23 @@ function plotSeries(plotSVG, model, axes, series, i){
         .enter()
         .append('rect')
         .attr('class', function (d){return 'column column--'  + series.className + (d.value < 0 ? ' negative' : ' positive');})
-        .attr('data-value', function (d){	return d.value;})
-		.attr('x', function (d){return axes.timeScale(d.key) + ((axes.timeScale.rangeBand() / model.y.series.length) * i);}) //adjust the x position based on the series number
-		//.attr("x", function(d) { return x1(d.key); })
-		.attr('y', function (d){return axes.valueScale(Math.max(0, d.value));})
-        .attr('height', function (d){return 	Math.abs(axes.valueScale(d.value) - axes.valueScale(0));})
-		//.attr("width", x1.rangeBand())
-        .attr('width', axes.timeScale.rangeBand() / model.y.series.length); //width is divided by series length
+        .attr('data-value', function (d){return d.value;})
+		.attr('x', function (d){
+			var adjustmentIfMultipleSeries = model.chartSubtype === 'multiple' ? ((axes.timeScale.rangeBand() / model.y.series.length) * i) : 0;
+			return axes.timeScale(d.key) + adjustmentIfMultipleSeries;
+		})
+		.attr('y', function (d, j){
+			var stackY = model.stackSeries({stack:j, value:d.value}, 0); //all series are now stacks
+			return axes.valueScale(Math.max(0, stackY));
+		})
+        .attr('height', function (d){return Math.abs(axes.valueScale(d.value) - axes.valueScale(0));})
+        .attr('width', function(){
+        	var adjustmentIfMultipleSeries = model.chartSubtype === 'multiple' ? model.y.series.length : 1;
+			return axes.timeScale.rangeBand() / adjustmentIfMultipleSeries;
```

Comment:
I was thinking here something along the lines might more readable:

From:
```Javascript
.attr('width', function(){
        	var adjustmentIfMultipleSeries = model.chartSubtype === 'multiple' ? model.y.series.length : 1;
			return axes.timeScale.rangeBand() / adjustmentIfMultipleSeries;
```
To:
```Javascript
.attr('width', function(){
      return adjustRangeToWidth(axes.timeScale.rangeBand());
}
```

with a function somewhere:

```Javascript
function adjustRangeToWidth(rangeBand){
    if (model.chartSubtype === 'multiple') {
         //on multiple series all columns must be squashed to fit in the same space 
         return rangeBand / model.y.series.length;    
    } else {
         return rangeBand; 
    }       
}
```